spec/integration/base_spec.rb

```diff
@@ -191,4 +191,46 @@ class ExampleBase < ActiveFedora::Base
       end
     end
   end
+
+  describe "nested AT::Resources" do
```

Comment:
I'd prefer this in it's own file or `spec/integration/nested_attributes_spec.rb`